prowler/lib/outputs/jira/jira.py

```diff
@@ -157,6 +174,10 @@ def cloud_id(self, value):
     def scopes(self):
         return self._scopes
 
+    @property
+    def using_basic_auth(self):
+        return self._using_basic_auth
```

Comment:
Is this property needed?

Reply:
I'm using it on the `test_connection`method